src/ontology/mondo-edit.obo

```diff
@@ -618478,6 +618482,7 @@ def: "A form of Cushing syndrome caused by prolonged exposure or excessive use o
 synonym: "exogenous Cushing’s syndrome" EXACT [PMID:25945066]
 synonym: "iatrogenic Cushing's syndrome" EXACT [PMID:25945066]
 is_a: MONDO:0018912 {source="PMID:25945066"} ! Cushing syndrome
+relationship: excluded_subClassOf MONDO:0005495 {source="Orphanet:553", source="https://orcid.org/0000-0002-7638-4659"} ! adrenal gland disorder
```

Comment:
@MeeSiing 
This was added to the wrong term. It should have been on MONDO:0018912